UI/src/app/models/comment.ts

```diff
@@ -0,0 +1,37 @@
+export interface IComment {
+  id: number;
+  text: string;
+  date: Date;
+  hasBeenEdited: boolean;
+
+  updateDate(): void;
+  setAsEdited(): void;
+  newValue(value: string): void;
+}
+
+export class Comment {
+  id: number;
+  date: Date;
+  hasBeenEdited: boolean;
+
+  constructor(
+    public text: string
```

Comment:
Do we need explicit ```public``` ?

Reply:
otherwise how would we create a property "text" here?

Reply:
The same way as ```id```